### PR TITLE
Chore/Modifications

### PR DESCRIPTION
**Pull request by borut-t**
`isAuthenticated` is now synchronous operation.

### CODE REVIEW

File: Sources/Auth/Apple/AppleAuthenticator.swift [hunk 1]
```diff
@@ -166,11 +173,16 @@ private extension AppleAuthenticator {
                                            name: ASAuthorizationAppleIDProvider.credentialRevokedNotification,
                                            object: nil)
   }
+  
+  func rejectSignIn(with error: Error) {
+    storage.setValue(false, forKey: storageAuthenticatedKey)
+    processingPromise?.reject(with: error)
```
Comment (tonikocjan) [inline Sources/Auth/Apple/AppleAuthenticator.swift:179]: `processingPromise` should probably be deleted, right?

File: Sources/Auth/Core/SocialAuthenticationManager.swift [hunk 2]
```diff
@@ -27,8 +35,8 @@ extension SocialAuthenticationManager {
     authenticators.forEach { $0.signOut() }
   }
   
-  public var isAuthenticated: PovioKitPromise.Promise<Authenticator.Authenticated> {
-    all(promises: authenticators.map { $0.isAuthenticated })
-      .map { $0.contains(true) }
+  public func canOpenUrl(_ url: URL, application: UIApplication, options: [UIApplication.OpenURLOptionsKey : Any]) -> Bool {
+    guard let activeAuthenticator = authenticators.first(where: { $0.isAuthenticated }) else { return false }
```
Comment (tonikocjan) [inline Sources/Auth/Core/SocialAuthenticationManager.swift:39]: You can use the computed property instead.

**Review by tonikocjan — APPROVED** [review]
🙌

File: Sources/Auth/Core/SocialAuthenticationManager.swift (hunk 2 above)
Comment (borut-t) [reply Sources/Auth/Core/SocialAuthenticationManager.swift:39]: Indeed 👍

**Pull request merged**